### ISSUE TITLE
JOSS paper: discussion of GPIB support

### ISSUE THREAD

**Issue by MatthieuDartiailh** [SwiftVISA/SwiftVISASwift#7]
The "current limitation and future work" section discusses USB, VXI-11 (misspelled in VX11)  but does not address GPIB support. While I assume PCIe card support make little sense, GPIB adapters could be supported through USB-adapters.

https://github.com/openjournals/joss-reviews/issues/4752

**Comment by OHildreth** [comment]
**Authors’ Response to Reviewer Comment**
Thanks.  We have fixed the VXI-11 misspelling and updated our text to discuss the use of GPIB adapters.

**Original Text**
There are no plans in `SwiftVISASwift` to directly support `FirewireInstrument` given that this legacy connection is not in widespread use.

**Updated Text**
There are no plans in `SwiftVISASwift` to directly support `GPIBInstrument` or `FirewireInstrument` given that these legacy connections re not in widespread use.  However, we have used GPIB-to-USB and GPIB-to-TCIIP adapters[@prologix] to control our instruments with SwiftVISA` and SwiftVISASwift` respectively.  These adapters were chosen because they don’t require additional drivers, making them platform independent.

**Comment by MatthieuDartiailh** [comment]
Saying that GPIB is not widely used sounds extremely weird to me given the number of old instruments I met in different labs that do use it and the number of people coming to PyVISA with GPIB specific applications so it may be worth toning down a bit the text. 

Regarding adapters it may be worth clarifying what was one and what is envisoned. Prologix adapter are weird since they deviate from the standard behavior of GPIB: you talk to the adapter which then translate for you. It may make sense for you to not want to go further but it may be worth to write explicitly.

**Comment by OHildreth** [comment]
Authors’ Response to Reviewer Comment
Thanks.  We have updated the text to reflect that GPIB is still in use, but that adapters exist 

**Original Text**
There are no plans in `SwiftVISASwift` to directly support `GPIBInstrument` or `FirewireInstrument` given that these legacy connections not in widespread use.  However, we have used GPIB-to-USB and GPIB-to-TCIIP adapters[@prologix] to control our instruments with SwiftVISA` and SwiftVISASwift` respectively.  These adapters were chosen because they don’t require additional drivers, making them platform independent.

**Updated Text**
We currently have no plans to update the `SwiftVISA` repositories to directly support `GPIBInstrument` or `FirewireInstrument`; however, contributors are welcome to submit pull-requests to add this functionality.  For our own projects, we use GPIB-to-USB and GPIB-to-TCP/IP adapters[@prologix] by Prologix to control our instruments with `SwiftVISA` and `SwiftVISASwift`.  These adapters work well with `SwiftVISA` and `SwiftVISASwift` because they don’t require installation of additional drivers and because the adapter internally translates the GPIB commands for you.
